### PR TITLE
CLI tool for creating an extSource

### PR DESCRIPTION
**Pull request by tauceti2**

### CODE REVIEW

File: perun-cli/addExtSource [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+use Getopt::Long qw(:config no_ignore_case);
+use Perun::Agent;
+use Perun::Common qw(printMessage);
+
+sub help {
+	return qq{
+	Adds an external source. External source name and type are required fields.
+
+	Example of Supported external source types. 
+	For full list see https://github.com/tauceti2/perun/tree/master/perun-core/src/main/java/cz/metacentrum/perun/core/impl
```
Comment (licehammer) [inline perun-cli/addExtSource]:
```suggestion
	For full list see https://github.com/CESNET/perun/tree/master/perun-core/src/main/java/cz/metacentrum/perun/core/impl
```

Comment (tauceti2) [reply perun-cli/addExtSource]: Fixed, thanks.

**Comment by stavamichal** [comment]
There is already cli script createExtSource. Is there any reason for this new one?

**Comment by stavamichal** [comment]
Closing because of duplicity.

**Pull request closed without merging**